.github/workflows/release.yml

```
@@ -156,7 +157,39 @@ jobs:
         with:
           registry: us-docker.pkg.dev
           username: oauth2accesstoken
-          password: ${{ steps.auth.outputs.auth_token }}
+          password: ${{ steps.auth.outputs.access_token }}
 
       - name: Publish web-sdk-demos to us-docker.pkg.dev
         run: docker push us-docker.pkg.dev/optable-artifact-registry/optable/optable-web-sdk-demos:${{ github.ref_name }}
+
+  slack-notification:
+    needs: [tests-prettier, build, deploy-sdk-to-npm, define-gcs-versions-to-update, deploy-sdk-to-gcs, deploy-demo]
+    runs-on: ubuntu-22.04
+    if: ${{ failure() }}
+    steps:
+      - name: Post to Slack
+        uses: slackapi/slack-github-action@v2.0.0
+        with:
+          method: chat.postMessage
+          token: "${{ secrets.SLACK_MESSENGER_APP_TOKEN }}"
+          payload: |
+            channel: "${{ vars.ALERTS_SLACK_CHANNEL_ID }}"
```

Since this is a public repo I added ALERTS_SLACK_CHANNEL_ID  as secret instead of vars
```suggestion
            channel: "${{ secrets.ALERTS_SLACK_CHANNEL_ID }}"
```